packages/account-export/src/lib/components/Passphrase.tsx

```diff
@@ -18,16 +19,26 @@ export const Passphrase: React.FC<PassphraseProps> = ({
   setHasCopied,
   onCloseModal,
   onBack,
+  onPassphraseSave,
 }) => {
-  // TODO: reserve this secret key and pass down to next step for encryption for WEP-213
-  const secretKey = generator.generate({
-    length: 10,
-    numbers: true,
-    strict: true,
-    lowercase: true,
-    uppercase: true,
-    symbols: true,
-  });
+  const [secretKey, setSecretKey] = useState("");
+
+  useEffect(() => {
+    const key = generator.generate({
```

Comment:
Could this go in `helpers/importAccount` instead? It would be nice to consolidate all methods involved in account encryption into a single module.